src/routes/+layout.svelte

```diff
@@ -57,6 +57,8 @@
   </div>
 {/if}
 
+<ModeWatcher />
```

Comment:
Remove for this PR to avoid accidental dark mode switches from lingering cookies via other forward looking branches